### DISCUSSION TITLE
How to initialize a repo in an empty directory?

### DISCUSSION THREAD

**Discussion by jkbellew** [jesseduffield/lazygit#2021] [Q&A] [answered]
Like the title I receive the prompt, "Not in a git repository. Create a new git repository? (y/n):" when I'm in an empty directory that hasn't been initialized. I answer y but nothing happens. It does not initialize the directory and I have to drop back down into the command line and issue 'git init'. 

I'm on Windows but I don't know why that would be an issue, but I'm letting that be known now instead of later.

Any assistance or answer is appreciated.

**Comment by mark2185** [comment, marked as answer]
See #1940 

Either build from source or wait for the new release.
